addons/http_routing/models/ir_http.py

```diff
@@ -456,7 +456,7 @@ def _dispatch(cls):
 
         # For website routes (only), add website params on `request`
         if request.is_frontend:
-            request.redirect = lambda url, code=302: werkzeug.utils.redirect(url_for(url), code)
+            request.redirect = lambda url, code=302: http.redirect(url_for(url), code)
```

Comment:
As discussed, this makes me think that we should perhaps expand this to expose `request.redirect()` method directly in WebRequest, and override it for frontend requests.

This could have some advantages:
- you could use `request.redirect()` as a shortcut without any extra import, as `request` is often already imported
- it would present a consistent API for those cases where we currently want to use `request.redirect()` in front-end requests

You could go further, and have `http.redirect()` be only a helper that calls the request's `redirect()` method when a `request` is defined.

Or go *even* further, and have `request.redirect()` call `ir.http._redirect()` when available (`if request and request.db`), to allow inheritance of the redirection behavior, instead of the lambda monkey-patching done here. A bit like what we do for the exception handling.

But maybe I'm going too far :)

Reply:
> it would present a consistent API for those cases where we currently want to use request.redirect() in front-end requests

I push force in main commit 

> Or go even further ...

I try to do you last request in a new commit, not sure 100% it is what you was expected...
Let me know if I misunderstand your request.